pkg/operator/ceph/pool/config.go

```diff
@@ -94,3 +102,37 @@ func generateStatusInfo(p *cephv1.CephBlockPool) map[string]string {
 	m[RBDMirrorBootstrapPeerSecretName] = buildBoostrapPeerSecretName(p.Name)
 	return m
 }
+
+func (r *ReconcileCephBlockPool) expandBootstrapPeerToken(cephBlockPool *cephv1.CephBlockPool, namespacedName types.NamespacedName, token []byte) ([]byte, error) {
```

Comment:
This only needs to be done once, right? So we can add a comment to remove it in the next minor release?

Reply:
No this will always be needed :)

Reply:
Should we persist it somewhere so we don't have to keep expanding it? Or it probably is simpler just to expand it every time.

Reply:
Yes, I think expanding everytime is fine too.